pages/toDOH.jsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from "react";
+import Header from "../components/Header";
+import ToDOHModal from "../components/Modal";
+import { IconButton } from "@chakra-ui/button";
+import { Flex } from "@chakra-ui/layout";
+import { SunIcon, PhoneIcon } from "@chakra-ui/icons";
+
+const toDOHs = [
+  {
+    id: 1,
+    name: "Male das Pony",
+    inside: true,
+    description1:
+      "Hol dir ein Blatt Papier und einen Stift und male ein Pony, das eine Möhre Kaut und einen Hut auf hat",
+    description2: "Setze deinen Handy-Timer auf 15 Minuten",
+  },
+  {
+    id: 2,
+    name: "Putze dein Küchenfenster",
+    inside: true,
+    description1:
+      "Schnapp dir einen Lappen, Glasspiritus und einen Abzieher und gönn dir den Putz",
+    description2: "Setze deinen Handy-Timer auf 15 Minuten",
+  },
+  {
+    id: 3,
+    name: "Hüpf nach draußen und bringe drei kleine Blumen wieder mit",
+    inside: false,
+    description1:
+      "Pack eine Gartenschere ein und suche draußen 3 hübsche Zweige oder Blümchen",
+    description2:
+      "Setze deinen Handy-Timer auf 15 Minuten und pack die armen Blumen in die Vase!",
+  },
+];
+
+function YourRandomToDOH({ inside, id }) {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [isInsideToDOH, setIsInsideToDOH] = useState();
+
+  const getRandomToDOH = (items) => {
+    const randomItemNr = Math.floor(Math.random() * items.length);
+    return items[randomItemNr];
+  };
+
+  const insideToDOHs = (toDOHs) => {
+    return toDOHs.filter((toDOH) => toDOH.inside);
+  };
+
+  const outsideToDOHs = (toDOHs) => {
+    return toDOHs.filter((toDOH) => !toDOH.inside);
+  };
```

Comment:
Since ToDOHs are global, there is no need to pass them to the functions here. They are in a parent scope and thus "inherited".

```suggestion
  const insideToDOHs = () => {
    return toDOHs.filter((toDOH) => toDOH.inside);
  };

  const outsideToDOHs = () => {
    return toDOHs.filter((toDOH) => !toDOH.inside);
  };
```

You could make use of destructuring to make the code briefer:

```suggestion
  const insideToDOHs = () => {
    return toDOHs.filter(({inside}) => inside);
  };

  const outsideToDOHs = () => {
    return toDOHs.filter(({inside}) => !inside);
  };
```

Since toDOHs are not changing during one render you also could get rid of the functions completely and just calculate the filtered arrays upfront:

```suggestion
  const insideToDOHs =  toDOHs.filter(({inside}) => inside);
  const outsideToDOHs =  toDOHs.filter(({inside}) => !inside);
```

I guess that you'll at some point have them passed to the component via prop or in some state.
Then this could be the final form.

If you're keeping the data as a static variable you could calculate `insideToDOHs` and `outsideToDOHs` out side of the component function. Thus reducing the stuff that has to be recalculated on every render.